cli-plugins/socket/socket_nodarwin.go

```diff
@@ -15,5 +15,6 @@ func listen(socketname string) (*net.UnixListener, error) {
 
 func onAccept(conn *net.UnixConn, listener *net.UnixListener) {
 	// do nothing
-	// while on darwin we would unlink here; on non-darwin the socket is abstract and not present on the filesystem
+	// while on darwin and OpenBSD we would unlink here;
+    // on non-darwin the socket is abstract and not present on the filesystem
```

Comment:
Looks like the indentation is off here (and using spaces, instead of tabs) 😅

Reply:
Thanks! fixed.